packages/block-editor/src/components/fullscreen-mode-close/index.js

```diff
@@ -18,30 +18,31 @@ const wordPressLogo = (
 	</SVG>
 );
 
-function FullscreenModeClose() {
-	const { isActive, postType } = useSelect( ( select ) => {
+function FullscreenModeClose( { isActive } ) {
```

Comment:
The behavior of that button look completely different between the two editors. Do you think we should just copy/paste addapt?

 - I see the `core/editor` store  is  used in block-editor which is not great.
 - isActive has been extractedd
 - the target URL is different.

Maybe it does make sense as a small component in the upcoming `@wordpress/interface` package but I wonder if it's a too early abstraction for now.

Reply:
> The behavior of that button look completely different between the two editors. Do you think we should just copy/paste addapt?

Yes, I think it might make more sense to do that for now. We can consider abstracting it in `@wordpress/interface` later on if needed.

Reply:
Refactored as suggested in 488fa35bbd273148c18b43098b9e9287538c3c2c.